style/search.css

```diff
@@ -0,0 +1,53 @@
+#geocoder-search-panel {
+  position: absolute;
+  top: 8px;
+  left: 48px;
+  z-index: 100;
+  padding: 3px;
+}
+#geocoder-search-input {
+  border: 1px solid lightgray;
+  font-size: 1em;
+  padding: 3px;
+}
+#geocoder-live-results {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+.gc-result-item {
+  border: 1px solid lightgray;
+  border-color: lightgray;
+  border-width: 1px 1px 0px 1px;
+  background-color: white;
+  padding: 2px;
+  cursor: pointer;
+}
+.gc-result-item:last-child {
+  border: 1px solid lightgray;
+  background-color: white;
+  padding: 2px;
+}
+.gc-result-item-selected {
+  border: 1px solid lightgray;
+  border-color: lightgray;
+  border-width: 1px 1px 0px 1px;
+  background-color: lightgrey;
+  padding: 2px;
+  cursor: pointer;
+}
```

Comment:
Suggest having `.gc-result-item` on always and toggling the class `.gc-result-item-selected` only the values to change (background-color).